scripts/build.js

```diff
@@ -102,6 +103,7 @@ let spec = await dereferenceDocument(doc);
 
 spec.components = {};
 
+// Merge instances of `allOf` in methods.
 function recursiveMerge(schema) {
   schema = merger(schema);
```

Comment:
Unrelated, but it looks like `merger` is [recursive](https://github.com/mokkabonna/json-schema-merge-allof#options) by default. I checked against a simple test case involving `oneOf`, and it seems to work. Does this function address some corner case?